--- busybox-1.17.0/networking/udhcp/common.c	2010-07-07 14:00:55.845283070 +0200
+++ busybox-1.17.0-sali/networking/udhcp/common.c	2010-07-07 14:03:42.823841626 +0200
@@ -20,6 +20,7 @@
  */
 const struct dhcp_optflag dhcp_optflags[] = {
 	/* flags                                    code */
+        { OPTION_IP                   | OPTION_REQ, 0xe0 }, /* SALI_IMAGESERVER   */
 	{ OPTION_IP                   | OPTION_REQ, 0x01 }, /* DHCP_SUBNET        */
 	{ OPTION_S32                              , 0x02 }, /* DHCP_TIME_OFFSET   */
 	{ OPTION_IP | OPTION_LIST     | OPTION_REQ, 0x03 }, /* DHCP_ROUTER        */
@@ -78,6 +79,7 @@
  */
 /* Must match dhcp_optflags[] order */
 const char dhcp_option_strings[] ALIGN1 =
+	"saliimgsrv" "\0"       /* SALI_IMAGESERVER    */
 	"subnet" "\0"      /* DHCP_SUBNET         */
 	"timezone" "\0"    /* DHCP_TIME_OFFSET    */
 	"router" "\0"      /* DHCP_ROUTER         */
--- busybox-1.17.0/networking/udhcp/common.h	2010-07-07 14:01:18.965351097 +0200
+++ busybox-1.17.0-sali/networking/udhcp/common.h	2010-07-07 14:04:39.431130842 +0200
@@ -105,6 +105,7 @@
  * Commented out options are handled by common option machinery,
  * uncommented ones have spacial cases (grep for them to see).
  */
+#define SALI_IMAGESERVER        0xe0
 #define DHCP_PADDING            0x00
 #define DHCP_SUBNET             0x01
 //#define DHCP_TIME_OFFSET      0x02 /* (localtime - UTC_time) in seconds. signed */
